gui/wxpython/core/gcmd.py

```diff
@@ -26,17 +26,24 @@
 """
 
 import os
+import wx
 import sys
 import time
 import errno
 import signal
-import traceback
 import locale
+import traceback
 import subprocess
+
 from threading import Thread
-import wx
+from core.debug import Debug
+from core.globalvar import SCT_EXT
+
+from grass.script import core as grass
```

Comment:
Even if that's not from this PR, this doesn't sound right, and quite ambiguous (the alias's name)

Reply:
I've noticed that across our codebase, there is a consistent use of the alias grass for the core module. Given that this pattern is present in around 51 files, it might be worth discussing whether we want to maintain this convention or consider a more descriptive alias.

Reply:
I'm pretty sure this was introduced when transitioning form a single-file package called grass(.script?) to multiple files (modules). Functions in original grass or something like that were moved core and grass or grass.script got new functions. _grass.script.core_ then became what _grass_ once was. These imports were the least intrusive change, so it was implemented that way. This was in the pre-v7-times on Subversion trunk, but it was propagated sufficiently through the code base at the time of v7 release that it became a standard for v7. We came up with `grass.script as gs` as a better practice, but didn't go back (yet) to replace all occurrences of `core as grass`.